javascript/ql/src/Statements/UselessRangeCheck.ql

```diff
@@ -0,0 +1,59 @@
+/**
+ * @name Useless range check
+ * @description If a range check always fails or always succeeds it is indicative of a bug.
```

Comment:
Comma missing?
If a range check always fails or always succeeds, it is indicative of a bug.